### DISCUSSION TITLE
step by step installation of OCR

### DISCUSSION THREAD

**Discussion by ismetsener** [pimcore/pimcore#9423] [General]
Hi,

These 2 discussions were related to OCR:
https://github.com/pimcore/pimcore/issues/662
https://github.com/pimcore/pimcore/issues/179

But it not clear to me how install and use OCR

I like to upload a PDF  File  (invoice)
after I like to read all the characters and put all the invoice lines (qty, product, price) in a table

here is a example of what I want:
https://docparser.com/blog/invoice-scanning/?cli_action=1623589747.638

what are the steps to realize this?

kind regards,
Ismet

**Comment by jdreesen** [comment]
Can you please stop screaming in your issue/discussion titles?
